sql_metadb/derived_tables/po_notes.sql

```diff
@@ -0,0 +1,33 @@
+--metadb:table po_notes
+--Creates a derived table to show all notes attached to purchase orders, including general notes, renewal notes, and notes when purchase order is closed. 
+
+DROP TABLE IF EXISTS po_notes;
+CREATE TABLE po_notes AS
+SELECT
+po.id AS po_id,
+(po.jsonb -> 'metadata' ->> 'createdDate')::timestamptz AS created_date,
+po.jsonb ->> 'poNumber' AS po_number,
+po.jsonb ->> 'workflowStatus' AS po_workflow_status,
+po_notes.jsonb #>> '{}' AS po_note,
+po_notes.ORDINALITY AS po_note_ordinality,
+po.jsonb -> 'closeReason' ->> 'note' AS po_close_reason_note,
+po.jsonb -> 'ongoing' ->> 'renewalNote' AS po_renewal_note
```

Comment:
I could not find renewalNote in any databases or under "ongoing" in the JSON schema here: https://s3.amazonaws.com/foliodocs/api/mod-orders-storage/p/purchase-order.html

Did I look in the wrong place?  Are you sure this is correct?

Reply:
it is on po_line table: https://s3.amazonaws.com/foliodocs/api/mod-orders-storage/p/po-line.html#orders_storage_po_lines__id__get 
and it is extracted in the transformed table.@arthur-aguilera please deleted "renewal note" from this derived table.